charmap.asm

```diff
@@ -281,7 +281,9 @@
 
 	charmap "「", $70
 	charmap "」", $71
+	charmap "『", $72
```

Comment:
`"『"` is already mapped under `; Actual characters (from gfx/font/font_battle_extra.png)`.

Reply:
![](https://i.imgur.com/9zA8xky.png)
This is the character from a different font file, so it feels weird to include all the way up there, but I guess it makes sense.

Reply:
font_extra.png has English curly quotes and middle dot; font_battle_extra.png has 『 and ID No.